### DISCUSSION TITLE
PSBT using node js

### DISCUSSION THREAD

**Discussion by rohitkeshav27** [blockstack/stacks-wallet-web#4752] [Q&A]
Can anybody help in creating a PSBT transaction in the leather wallet using code?

**Comment by kyranjamie** [comment]
Can you elaborate what you're trying to achieve with the PSBT? Are you familiar with the [`bitcoinjs-lib`](https://github.com/bitcoinjs/bitcoinjs-lib) and [`@scure/btc-signer`](https://github.com/paulmillr/scure-btc-signer) libraries? The documentation for both this libraries can help.

**Comment by rohitkeshav27** [comment]
Hello ,
Was able to figure out that , but can you let me know how the total fee is
calculated if I give the sats/vb
